docs/_templates_overwrite/index.html

```diff
@@ -1,568 +1,680 @@
-					</ul>
-                    <div class="wow fadeIn" data-wow-duration="1s">
+                                    <pre class="code">docker run -it --rm xonsh/xonsh:slim</pre>
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class="card">
+                            <div class="card-header" role="tab" id="faq4">
+                                <h5 class="mb-0">
+                                    <a class="collapsed" data-toggle="collapse" href="#collapse5" aria-expanded="false" aria-controls="collapse5">Platform package managers: brew, apt, pacman, dnf</a>
+                                </h5>
+                            </div>
+                            <div id="collapse5" class="collapse" role="tabpanel" aria-labelledby="faq5" data-parent="#accordion">
+                                <div class="card-body">
+                                    <p>OSX brew:</p><pre class="code">brew install xonsh</pre>
+                                    <p>Debian/Ubuntu apt:</p><pre class="code">apt install xonsh</pre>
+                                    <p>Arch Linux pacman:</p><pre class="code">pacman -S xonsh</pre>
+                                    <p>Fedore dnf:</p><pre class="code">dnf install xonsh</pre>
+                                </div>
+                            </div>
+                        </div>
+
+                    </div>
+                </div>
+
+            </div>
+
+            <div class="text-center">
+                <p style="margin-top: 30px;">
+                    <a href="https://xon.sh/contents.html#installation" target="_blank"> <i class="icofont-ui-note"></i> More installation methods →</a>
+                </p>
+            </div>
+
+        </div>
+    </section>
+
+    <section id="product-area" data-scroll-index="4">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-6 offset-lg-3 col-md-8 offset-md-2">
+                    <div class="section-heading text-center">
+                        <h2>Xontribs</h2>
+                        <p>Xontributions, or xontribs, are a set of tools and conventions for extending the functionality of xonsh. </p>
+                    </div>
+                </div>
+            </div>
+            <div class="row" id="github_xontribs">
+                <div class="col-lg-3 col-md-6">
+                    <div class="product-single text-center">
+                        <div class="prod-details">
+                            <p>
+                            <a href="https://github.com/anki-code/xontrib-pipeliner" target="_blank">pipeliner</a>
+                            </p>
+                            <p>Let your pipe lines from the standard output flow thru the Python code in the xonsh shell. </p>
+                        </div>
+                    </div>
+                </div>
```

Comment:
I see the auto-updating version on your landing page demo -- I think a bunch of stuff here can be removed now.